src/build_platform/local_docker.rs

```diff
@@ -73,7 +73,7 @@ impl LocalDocker {
         }
 
         // arbitrary percentage that should make the job anytime
-        const DISK_FREE_SPACE_PERCENTAGE_BEFORE_PURGE: u64 = 20;
+        const DISK_FREE_SPACE_PERCENTAGE_BEFORE_PURGE: u64 = 40;
```

Comment:
u64 -> u8

Reply:
just to avoid casting on other functions from libs returning u64